app-service-template/README.md

```diff
@@ -0,0 +1,41 @@
+# Application Service Template
+
+This folder contains a buildable/runnable template for a new custom application service based on the Hanoi 1.3.1 release of the App Functions SDK. 
```

Comment:
should this be updated to 2.0 to match go.mod?

Reply:
Yep, should be 2.x and when we release it and the go.mod need to be update to the actual version.